.eslintrc

```diff
@@ -18,8 +18,13 @@
       "files": [ "**/*.ts" ],
       "extends": [
         "eslint:recommended",
-        "prettier"
-      ]
+        "prettier",
```

Comment:
prettier is already defined

Reply:
The difference is the comma, to allow additional options.